docs/usage/quickstart.rst

```diff
@@ -3,85 +3,119 @@
----------------------
-To tell the Ansible_-based deployment system on which machines MetalK8s should
-be installed, a so-called *inventory* needs to be provided. This inventory
-contains a file listing all the hosts comprising the cluster, as well as some
-configuration.
+Clone or Copy the MetalK8s Git Repo
+-----------------------------------
+
+Clone the MetalK8s project from GitHub:
+
+.. code-block:: shell
+
+   $ git clone https://github.com/scality/metal-k8s
+
+Define an Inventory
+-------------------
+
+To tell the Ansible_-based deployment system which machines MetalK8s shall be
+installed on, you must provide an *inventory*. This inventory contains a file
+that lists all hosts in the cluster, as well as some configuration.
 
 .. _Ansible: https://www.ansible.com
 
-First, create a directory, e.g. :file:`inventory/quickstart-cluster`, in which
-the inventory will be stored. For our setup, we need to create two files. One
-listing all the hosts, aptly called :file:`hosts`:
+To create an inventory:
+
+  1. Create a directory inside MetalK8s (for example,
+     :file:`inventory/quickstart-cluster`) in which the inventory will
+     be stored.
+
+     .. code-block:: shell
+
+       $ cd metal-k8s
+       $ mkdir -p inventory/quickstart-cluster
+       $ cd inventory/quickstart-cluster/
 
-.. code-block:: ini
+  2. Create the :file:`hosts` file, which contains a listing of all hosts.
 
-    node-01 ansible_host=10.0.0.1 ansible_user=centos
-    node-02 ansible_host=10.0.0.2 ansible_user=centos
-    node-03 ansible_host=10.0.0.3 ansible_user=centos
+     .. code-block:: ini
 
-    [kube-master]
-    node-01
-    node-02
-    node-03
+        node-01 ansible_host=10.0.0.1 ansible_user=centos
+        node-02 ansible_host=10.0.0.2 ansible_user=centos
+        node-03 ansible_host=10.0.0.3 ansible_user=centos
 
-    [etcd]
-    node-01
-    node-02
-    node-03
+        [kube-master]
+        node-01
+        node-02
+        node-03
 
-    [kube-node]
-    node-01
-    node-02
-    node-03
+        [etcd]
+        node-01
+        node-02
+        node-03
 
-    [k8s-cluster:children]
-    kube-node
-    kube-master
+        [kube-node]
+        node-01
+        node-02
+        node-03
 
-Make sure to change IP-addresses, usernames etc. according to your
-infrastructure.
+        [k8s-cluster:children]
+        kube-node
+        kube-master
 
-In a second file, called :file:`kube-node.yml` in a :file:`group_vars`
-subdirectory of our inventory, we declare how to setup storage (in the
-default configuration) on hosts in the *kube-node* group, i.e. hosts on which
-Pods will be scheduled:
+    Change the IP addresses, host names, and user names to conform to
+    your infrastructure.
 
-.. code-block:: yaml
+  3. Create a :file:`group_vars` subdirectory in the inventory directory.
 
-    metal_k8s_lvm:
-      vgs:
-        kubevg:
-          drives: ['/dev/vdb']
+   - A second file, called :file:`kube-node.yml`, in :file:`group_vars`
```

Comment:
This subdivision with 2 ul's feels wrong. The fact there's a first li which talks about some 'second file', but doesn't *do* anything about that file, then a second li where that file is handled is bleh